storage/src/test/kotlin/maru/database/kv/KvDatabaseTest.kt

```diff
@@ -179,4 +179,100 @@ class KvDatabaseTest {
       assertThat(db.getSealedBeaconBlock(testBeaconBlock2Number)).isNull()
     }
   }
+
+  @Test
```

Comment:
I created a ticket to use single testsuite for InMemory and the persistent implementations https://github.com/Consensys/maru/issues/198